library/datasort.c

```diff
@@ -576,6 +651,21 @@ static int datasort_split(struct datasort_cfg *dcfg)
 	for (n = 0; n < dcfg->bctl_cnt; ++n) {
 		assert(dcfg->bctl[n] != NULL);
 
+		// If base is already sorted we don't need split it into chunks and sort it.
+		// Instead of that we can just iterate it over in the merge phase.
+		if (datasort_base_is_sorted(dcfg->bctl[n])) {
+			EBLOB_WARNX(dcfg->log, EBLOB_LOG_ERROR, "defrag: datasort_add_view_chunk: start, index %d",
```

Comment:
Why does it have `EBLOB_LOG_ERROR`?

Reply:
I understood your point about the size of logs in a way that EBLOB_LOG_INFO should be less frequent. Could you clarify one more time what do you want here?

Reply:
Eblob has follow levels:

- `EBLOB_LOG_ERROR` - log about some error or failure
- `EBLOB_LOG_INFO` - log with general useful information but less verbose
- `EBLOB_LOG_NOTICE` - log with general useful information and more verbose
- `EBLOB_LOG_DEBUG` / `EBLOB_LOG_SPAM` - most verbose log